src/Init/Init_Load_FlagCriteria.cpp

```diff
@@ -195,6 +198,10 @@ void Init_Load_FlagCriteria()
 //          OPT__FLAG_NPAR_PATCH/CELL load integers
             else if ( FlagMode == 6 )  sscanf( input_line, "%d%d",  &Trash, &FlagTable_NParPatch[lv] );
             else if ( FlagMode == 7 )  sscanf( input_line, "%d%d",  &Trash, &FlagTable_NParCell [lv] );
+//          OPT__FLAG_ANGULAR has four columns to be loaded
+            else if ( FlagMode == 14 ) sscanf( input_line, "%d%lf%lf%lf", &Trash, &FlagTable_Angular[lv][0],
+                                                                                  &FlagTable_Angular[lv][1],
+                                                                                  &FlagTable_Angular[lv][2] );
```

Comment:
Should we check whether `AngRes_Max < AngRes_Min` if both are given, and raise a warning/error here or somewhere?